dactyl-config.yml

```
@@ -1612,9 +1612,60 @@ pages:
 #             - en
 #             - ja
 
-    -   name: Manage the rippled Server
-        html: manage-the-rippled-server.html
+        # TODO: translate title & blurb
+    -   name: Interoperability - EVM Sidechain
+        html: evm-sidechains.html
         parent: tutorials.html
+        blurb: Learn how to interact with the EVM Sidechain Devnet.
+        template: pagetype-category.html.jinja
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/get-started-evm-sidechain.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/connect-metamask-to-xrpl-evm-sidechain.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/join-evm-sidechain-devnet.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/evm-sidechain-validator-security.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/evm-sidechain-run-a-validator-node.md
+        targets:
+            - en
+            - ja
+
+# Infrastructure --------------------------------------------------------------------
+
+    -   name: XRP Ledger Infrastructure
+        html: xrpl-infrastructure.html
+        parent: docs.html
+        top_nav_grouping: Article Types
+        template: pagetype-category.html.jinja
+        blurb: Learn how to set up, maintain, test, and troubleshoot the XRP Ledger.
+        targets:
+            - en
```

It seems to me that a Japanese page should be added to target.

```suggestion
            - en
            - ja
```